projects/optic/src/commands/capture/capture.ts

```diff
@@ -182,7 +183,12 @@ const getCaptureAction =
 
     // make requests
     console.log(trafficDirectory);
-    const requests = makeRequests(captureConfig.requests, proxyUrl);
+    const concurrency = captureConfig.config?.request_concurrency || 25;
```

Comment:
Should the default here be something lower like 5 or something? Requests should resolve fairly quickly so higher concurrency might not mean faster due to contention